benchmarks/benchmark_functions/default_settings.py

```diff
@@ -98,8 +99,7 @@ def _vqe_defaults(hyperparams):
 
 	options_dict = {'interface': interface, 'diff_method': diff_method, 'optimize': optimize}
 
-	# if device name is given, create device
 	if isinstance(device, str):
-		device = qml.device(device, wires=qubits)
+		device = qml.device(device, wires=4)
```

Comment:
I like the syntax more of defining the number of qubits above...it makes it easier to read off the settings from the start.

Reply:
But if you keep `wires=4`, can you delete `qubits=4` above? And what about `singles, doubles = qchem.excitations(2, 4)`?

Reply:
Sure, I will use `qubits` and `electrons` instead of numbers everywhere.